.github/workflows/linux.yml

```diff
@@ -407,14 +407,16 @@ jobs:
 
   ONNX_Runtime:
     name: ONNX Runtime Integration
-    if: fromJSON(needs.smart_ci.outputs.affected_components).ONNX_RT ||
-        fromJSON(needs.smart_ci.outputs.affected_components).ONNX_FE
+    # Enable back once https://github.com/microsoft/onnxruntime/pull/19184 is merged
```

Comment:
May we use custom fork with changes until PR isn't merged?

Reply:
discussed offline